### DISCUSSION TITLE
Heat Transfer between Cell

### DISCUSSION THREAD

**Discussion by JayYDG** [pybamm-team/liionpack#272] [Q&A]
Just wonder how the heat transfer is defined in the model?  Does the contact area is taken into consideration?

**Comment by TomTranter** [comment]
It's using the same lumped thermal model that PyBaMM uses with a heat transfer coefficient, ambient temperature and surface area for heat loss https://docs.pybamm.org/en/stable/source/api/models/submodels/thermal/lumped.html
